scripts/setup/install_doc_deps.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eux
+
+cargo install mdbook-graphviz
+DEBIAN_FRONTEND=noninteractive sudo apt-get install --no-install-recommends --yes graphviz
```

Comment:
nit: Can you please put this script under the ubuntu folder?

Reply:
Done